Binary_Tree.py

```diff
@@ -0,0 +1,125 @@
+from Node import Node
+
+"""
```

Comment:
Docstrings go just after the "def" line of the element they describe. See e.g. https://www.python.org/dev/peps/pep-0257/#one-line-docstrings.